test/Spec/GT/DB/Usage.hs

```diff
@@ -0,0 +1,45 @@
+module Spec.GT.DB.Usage where
+
+import Spec.GT.Prelude
+
+import Control.Monad.Logger
+import Data.Time.Clock
+import Database.Persist.Postgresql
+
+import GT.DB.Usage as Usage
+import GT.DB.Schema.Ride
+import GT.DB.Schema.Component
+import GT.DB.Schema.Bike
+
+spec :: Spec
+spec = do
+    describe "new" $ do
+        it "works" $ do
+            pendingWith "really need a better database testing story!!"
```

Comment:
this testing story is quite bad! it clobbers the `geartracker` database directly, which is definitely *not* okay. But implementing the database testing stuff is getting deferred to a future PR.